src/common/RunParams.cpp

```diff
@@ -651,24 +642,16 @@ void RunParams::printFeatureKernels(std::ostream& str) const
   for (int fid = 0; fid < NumFeatures; ++fid) {
     FeatureID tfid = static_cast<FeatureID>(fid);
     str << getFeatureName(tfid) << std::endl;
-    for (int kid = 0; kid < NumKernels; ++kid) {
-      KernelID tkid = static_cast<KernelID>(kid);
-///   RDH DISABLE COUPLE KERNEL
-      if (tkid != Apps_COUPLE) {
-         KernelBase* kern = getKernelObject(tkid, *this);
-         if ( kern->usesFeature(tfid) ) {
-           str << "\t" << getFullKernelName(tkid) << std::endl;
-         }
-         delete kern;
-      }
-    }  // loop over kernels
+
     str << std::endl;
   }  // loop over features
   str.flush();
 }
-
+// TODO for Kokkos Team:  Commenting function body, because this infrastructure 
```

Comment:
TODO:  @rhornung67 , @davidbeckingsale - commented lines here, 654-674, also represent important functionality for RAJAPerf Suite that has not yet been implemented for Kokkos.  Would you guys please help me work through integrating these bits?